README.md

```diff
@@ -78,6 +79,31 @@ Source:
   * https://github.com/Samsung/TizenRT/blob/master/build/configs/artik053/README.md#add-usb-device-rules
 
 
+### Set up ARTIK 530
+
+In case of the ARTIK 530 devices, the communication happens over `SSH` like RPI2. You need to check a few things before starting remote-test.
+First, you must install rpm package which is `openssh, python3, rsync` on the remote target. 
+([openssh](http://download.tizen.org/releases/previews/iot/preview1/tizen-4.0-unified_20171016.1/repos/standard/packages/armv7l/openssh-6.6p1-1.2.armv7l.rpm), 
+[libpython3](http://download.tizen.org/releases/previews/iot/preview1/tizen-4.0-base_20170929.1/repos/arm/packages/armv7l/libpython3_4m1_0-3.4.4-2.6.armv7l.rpm), 
```

Comment:
Can you clarify whether we can use python-2.7? I find that `libpython-2.7.8-2.7.armv7l.rpm` exists in the repo. It would be better to get `simpletester.py` managed in the same version of other py files.

Reply:
To install python-2.7, you need many packages with dependencies. For python3, only two packages need to be installed.